services/src/api/model/operators.rs

```diff
@@ -602,7 +602,7 @@ impl From<OgrSourceTimeFormat> for geoengine_operators::source::OgrSourceTimeFor
     }
 }
 
-#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize, ToSchema)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Deserialize, Serialize, ToSchema, ToSql, FromSql)]
```

Comment:
general question: Is it a good idea to use the API structs directly for storage in the DB? You also said you wanted to use the API types strictly in the handlers and then convert them to something else immediately.

Reply:
Yes, but currently, the API structs are (mostly) used to store them in the DB.

We should tackle this in another task as I don't want to change the API methods or internals other than the FromSql/ToSql part.

Shouldn't be much a hassle in the new task since the fields and everything has the same name.